packages/components/package.json

```diff
@@ -43,6 +44,7 @@
 		"react-dates": "^17.1.1",
 		"react-spring": "^8.0.20",
 		"rememo": "^3.0.0",
+		"styled-components": "^4.4.0",
```

Comment:
Do you know if this has any noticeable impact on bundle size?

Reply:
It's not tiny, but it's not terribly large either (e.g. moment.js)

It's currently at:
45.1kb, minified
16.4kb, minified + gzipped

https://bundlephobia.com/result?p=styled-components@4.4.0

The team is reducing the library size by a lot (and making it faster) in v5:
https://medium.com/styled-components/announcing-styled-components-v5-beast-mode-389747abd987

One of the requirements is that you must be using React 16.8, for hook support. Thankfully, it looks like Gutenberg + `@wordpress/components` does :)